app/index.html

```diff
@@ -32,16 +33,257 @@ <h1>I'm Amar Glad to see you!</h1>
+              <p>2015</p>
+            </div>
+          </div>
+          <div class="card-body">
+            <div class="card-description">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="stack-section">
+              <div class="stack-container">
+                <span class="tech-stack">HTML</span>
+              </div>
+              <div class="stack-container">
+                <span class="tech-stack">CSS</span>
+              </div>
+              <div class="stack-container">
+                <span class="tech-stack">JavaScript</span>
+              </div>
+            </div>
+          </div>
+          <div class="card-footer">
+            <a href="#" class="show-button">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card-container">
+          <div class="card-img">
+            <img src="./assets/pngs/jobs/job4.png" alt="" class="job-img">
+          </div>
+          <div class="card-title">
+            <h1 class="card-title-text">Multi-Post Stories</h1>
+          </div>
+          <div class="card-subtitle">
+            <div class="card-el el1">
+              <p>CANOPY</p>
+            </div>
+            <span class="dividor"></span>
+            <div class="card-el el2">
+              <p>Back End Dev</p>
+            </div>
+            <span class="dividor"></span>
+            <div class="card-el el2">
+              <p>2015</p>
+            </div>
+          </div>
+          <div class="card-body">
+            <div class="card-description">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="stack-section">
+              <div class="stack-container">
+                <span class="tech-stack">HTML</span>
+              </div>
+              <div class="stack-container">
+                <span class="tech-stack">CSS</span>
+              </div>
+              <div class="stack-container">
+                <span class="tech-stack">JavaScript</span>
+              </div>
+            </div>
+          </div>
+          <div class="card-footer">
+            <a href="#" class="show-button">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- jobs lists ends here -->
+    <!-- about section starts here -->
+    <section class="about-section">
+      <div class="about-title">
+        <h1>About Myself</h1>
+      </div>
+      <div class="card-description about-body">
+        <p>Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my
+          work and experience! If you like what you see and have a project you need coded don't hasitate to contact me.
+        </p>
+      </div>
+      <div class="banner-footer about-social">
+        <a href="#">LET'S CONNECT</a>
```

Comment:
Instead of using the <a> tag, I recommend using the h2 tag. The h2 tag is a great way to structure your content and make it more accessible for all users. It helps to provide a clear hierarchy of headings and subheadings, making it easier for users to navigate your content.